src/scene/Scale.js

```diff
@@ -162,17 +162,23 @@ function ordinal(scale, rng, group) {
     scale.rangeBands(rng, pad, outer);
   }
 
-  if (!scale.invert) {
-    scale.invert = function(x, y) {
+  prev.range = rng;
```

Comment:
`prev` is just an arbitrary object here, not a previous scale reference. We use it to determine if the scale should be identified (within the changeset) as updated. So this shouldn't be a problem.